kokoro/scripts/test/go_test.sh

```diff
@@ -194,7 +194,7 @@ ulimit -n 1000000
 
 # Set up some command line flags for "gotestsum".
 gotestsum_args=(
-  --packages=./...
+  --packages="go list ./... | grep -v /submodules/"
```

Comment:
Doesn't this need to be inside of `$(...)`?

Reply:
After seeing a test failure, i'm realizing this line may not be needed to update at all since AFAIU it used for integration tests.

Reply:
Confirmed that adding `submodules/go.mod` avoids any need to update `go test ./...` use.